wcomponents-examples/src/main/java/com/github/bordertech/wcomponents/examples/petstore/ConfirmOrderPanel.java

```diff
@@ -73,14 +73,12 @@ public ConfirmOrderPanel() {
 		// Address
 		fieldPanel = new FieldPanel();
 		WTextField addressLine1 = new WTextField();
-		addressLine1.setAccessibleText("Street address line 1");
 		addressLine1.setMandatory(true);
 		addressLine1.setMaxLength(60);
 		addressLine1.setBeanProperty("address.line1");
 		fieldPanel.add("Street address", addressLine1);
 
 		WTextField addressLine2 = new WTextField();
-		addressLine2.setAccessibleText("Street address line 2");

```

Comment:
The UI in this view is not compliant with our accessibility standards. Could you temporarily replace setAccessibleText with setToolTip and add a TODO to fix the a11y in the UI as a whole.